internal/provider/inventory_resource_test.go

```diff
@@ -176,6 +176,7 @@ func TestAccInventoryResource(t *testing.T) {
 	updatedDescription := "A test inventory"
 	updatedVariables := "{\"foo\": \"bar\"}"
 	resource.Test(t, resource.TestCase{
+
```

Comment:
```suggestion
```

It's helpful to keep unnecessary changes out of a PR. This is a pretty minimal PR at this point (which is good!) so having a stray blank line stands out. I don't feel strongly about this specific line but I do ask to clean up unintended changes.